clients/client-python/gravitino/utils/exceptions.py

```diff
@@ -1,8 +1,3 @@
-"""
-Copyright 2024 Datastrato Pvt Ltd.
-This software is licensed under the Apache License version 2.
-"""
-
```

Comment:
As you have removed the header, the Rat license check is likely to fail. You'll need to add the MIT header or add the file to the Rat exclusions.